<launch>

	<!--Load softwate configuration -->
	<include file="$(find candybot_v2)/run/software_config.launch" />
	
	
	<!-- Nodes running -->
	<node name="motion_head_controller" pkg="candybot_v2" type="motion_head_controller.py" output="screen"/>

</launch>
